contracts/contracts/UserReplicaSetManager.sol

```diff
@@ -36,7 +37,7 @@ contract UserReplicaSetManager is RegistryContract, SigningLogic {
         require(
             _registryAddress != address(0x00) &&
             _userFactoryRegistryKey.length != 0,
-            "requires non-zero _registryAddress"
+            "Requires non-zero _registryAddress"
```

Comment:
my comment for this was to change the require stmt to be "Requires non-zero _registryAddress & non-empty registryKey" lol. w/e, handle later